scripts/install.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+BINARY_NAME="universal-sierra-compiler"
+LOCAL_BIN="${HOME}/.local/bin"
+
+main () {
+  download_and_extract_binary
+
+  addBinaryToPath
+
+  echo "${BINARY_NAME} (${release_tag}) has been installed successfully."
+}
+
+download_and_extract_binary() {
+  repo="https://github.com/software-mansion/universal-sierra-compiler"
+  # Fetch the latest release tag from GitHub API
+  release_tag=$(curl -# -Ls -H 'Accept: application/json' "${repo}/releases/latest" | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
```

Comment:
nitpick

this regexp is a bit complicated, it would be much cleaner to use jq but I understand if you do not want to include another dependency, so I leave this up to you